test/services/group_links/group_unlink_service_test.rb

```diff
@@ -14,6 +14,16 @@ def setup
       assert_difference -> { NamespaceGroupLink.count } => -1 do
         GroupLinks::GroupUnlinkService.new(@user, namespace_group_link).execute
       end
+
+      assert_enqueued_emails 2
+      assert_enqueued_email_with GroupLinkMailer, :access_revoked_user_email,
+                                 args: [Member.user_emails(namespace_group_link.group),
+                                        namespace_group_link.group,
+                                        namespace_group_link.namespace]
+      assert_enqueued_email_with GroupLinkMailer, :access_revoked_manager_email,
+                                 args: [Member.manager_emails(namespace_group_link.group),
```

Comment:
All `Member.manager_emails(namespace_group_link.group)` can be updated to `Member.manager_emails(namespace_group_link.namespace)` in this test file

Reply:
afc64834a297ec27332b860346f0d0e66c2b927b